### DISCUSSION TITLE
What’s the best order for the various calibration tests?

### DISCUSSION THREAD

**Discussion by flodooby** [OrcaSlicer/OrcaSlicer#2476] [General]
Hello,

I am new to 3d printer. In what order should we do the calibrations below:

Flow Dynamics
Flow rate
Temperature with Temp tower
Max volumetric speed testPressure
Retractation
Tolerance
Max flow rate
VFA
....

**Comment by Wookbert** [comment]
@flodooby Please change the title to **„What’s the best order for the various calibration tests?“**, so the community sees that it’s actually a question, not an explanation.

I have the pretty much same question: What’s the best order for the various calibration tests? I assume they are depending on each other, aren’t they? Is the proper order as listed in the OrcaSlicer Calbration Drop Down Menu, so Temperature ➔ Flow Rate ➔ Pressure Advance ➔ Retraction? In the [OrcaSlicer Wiki](https://github.com/SoftFever/OrcaSlicer/wiki/Calibration) the order is unfortunately different again.

**Comment by LJKraus** [comment]
This question has been asked several times before, but I've never seen a definitive answer.  I've used the menu order with good results.  I often wonder if repeating the tests would result in further improvements. 
I'd think that a different flow rate or PA would have an effect on the temp tower results, but I've never taken the time to run the tests again.
